src/annotator/anchoring/range.js

```diff
@@ -0,0 +1,494 @@
+import $ from 'jquery';
+
+import { xpathFromNode, nodeFromXPath } from './xpath-evaluate';
+import {
+  getFirstTextNodeNotBefore,
+  getLastTextNodeUpTo,
+  getTextNodes,
+} from './xpath-util';
+
+const Range = {};
+export default Range;
```

Comment:
I believe the `Range` object here exists because this code pre-dated modules originally. A more idiomatic conversion would be to make `sniff`, `BrowserRange`, `SerializedRange` and `NormalizedRange` named exports (ie. `export function sniff(...)`) etc.

If you do this you'll run into an ESLint warning about using things before defining them. With `sniff` that is easy to fix by moving it to the bottom of the file. For the other cases you'll need to suppress some of the warnings due to the circular dependencies.

Reply:
I'll make the named export, but I'll still need to set the Range={} object locally because of the 3-way dependency of the Range* classes, unless there is a more appropriate way to do that?